openwisp_utils/__init__.py

```diff
@@ -1,18 +1,18 @@
-VERSION = (0, 4, 4, 'final')
+VERSION = (0, 4, 4, "final")
```

Comment:
Does black recommend using double quotes? 
( If it does, that sucks, I just converted double quotes to single quotes in controller at a lot of places! :joy: )

Reply:
This is the default. We can configure it to use single quotes :)